wgpu/src/image.rs

```diff
@@ -292,57 +362,55 @@ impl Pipeline {
                     mem::size_of::<Instance>() as u64,
                 );
 
-                {
-                    let mut render_pass = encoder.begin_render_pass(
-                        &wgpu::RenderPassDescriptor {
-                            color_attachments: &[
-                                wgpu::RenderPassColorAttachmentDescriptor {
-                                    attachment: target,
-                                    resolve_target: None,
-                                    load_op: wgpu::LoadOp::Load,
-                                    store_op: wgpu::StoreOp::Store,
-                                    clear_color: wgpu::Color {
-                                        r: 0.0,
-                                        g: 0.0,
-                                        b: 0.0,
-                                        a: 0.0,
-                                    },
+                let mut render_pass = encoder.begin_render_pass(
```

Comment:
One of the main advantages of using an atlas is that it should allow us to render many images in a single draw call.

We should upload all the instance data and create only one render pass. Then, bind the corresponding atlas and draw the instances.